dist/isoCurrency.js

```diff
@@ -894,8 +894,16 @@ angular.module('isoCurrency', ['isoCurrency.common'])
 			if (!currency) {
 				return amount;			
 			}
+
 			var fractionSize = (fraction === void 0) ? currency.fraction : fraction;
-			return $filter('currency')(amount, currency.symbol, fractionSize);
+	        var currencyFilter = $filter('currency');         
+
+	        if(amount < 0){
```

Comment:
But to answer you, Angular uses parenthesis by default which is less understandable than the negative symbol based on the feedback I have received so far. It is just too technical for the masses.